packages/thegraph-data-access/src/subgraph-client.ts

```diff
@@ -19,8 +19,8 @@ export class SubgraphClient implements StorageTypes.IIndexer {
   private graphql: GraphQLClient;
   public readonly endpoint: string;
 
-  private readonly DEFAULT_PAGE_SIZE = 10;
-  private readonly MAX_PAGE_SIZE = 100;
+  private readonly DEFAULT_PAGE_SIZE = 100;
```

Comment:
This still caps the requests at 100 requests, so if a wallet has more they won't be able to get the rest.

At least currently.

Reply:
Yes, but if they are encrypted it could cause performance issues. So I am attempting to accommodate for both use cases.

Reply:
It will definitely force clients to start using pagination